TemplateBicep/modules/iothub.bicep

```diff
@@ -0,0 +1,13 @@
+param name string
+param location string
+
+resource iotHub 'Microsoft.Devices/IotHubs@2021-07-02' = {
```

Comment:
Nit. not sure we need a module for this single resource

Reply:
For now, I will leave each sub-resource in own file. Let's revisit once we start working more with them.